src/config.rs

```diff
@@ -11,20 +12,68 @@ pub enum WalletKeys {
 
 #[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
 #[serde(rename_all = "kebab-case")]
-pub enum WalletKind {
+pub enum WalletKeyOld {
     #[serde(rename = "p2wpkh")]
     P2wpkh,
 }
 
 #[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
 #[serde(rename_all = "kebab-case")]
-pub struct Config {
+pub enum WalletKey {
```

Comment:
:question:  why did you decide not to put `#[serde(tag = "kind")]` here as in the issue?